modules/ossm-deploying-jaeger.adoc

```diff
@@ -0,0 +1,105 @@
+
+* *streaming* - The streaming strategy is designed to augment the production strategy by providing a streaming capability that effectively sits between the Collector and the backend storage (Elasticsearch). This provides the benefit of reducing the pressure on the backend storage, under high load situations, and enables other trace post-processing capabilities to tap into the real time span data directly from the streaming platform (https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html/using_amq_streams_on_openshift/index[AMQ Streams]/ https://kafka.apache.org/documentation/[Kafka]).
+
+[NOTE]
+====
+The streaming strategy requires an additional Red Hat subscription for AMQ Streams.
+====
+
+[id="ossm-deploying-jaeger-default_{context}"]
+== Default Jaeger deployment
+
+To use the default `allInOne` Jaeger deployment strategy set `spec:addons:jaeger:install:storage:type` to `Memory`
+. You can accept the defaults or specify additional configuration options under `install`.  If you do not specify Jaeger configuration options, the Control Plane will use the `allInOne` deployment strategy by default.
+
+.Control Plane default Jaeger parameters (Memory)
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  version: v2.0
+  tracing:
+    sampling: 10000
+    type: Jaeger
+  addons:
+    jaeger:
+      name: jaeger
+      install:
+        storage:
+          type: Memory
+----
+
+[id="ossm-deploying-jaeger-production_{context}"]
+== Production Jaeger deployment
+
+To use the `production` deployment strategy, set  `spec:addons:jaeger:install:storage:type` to 'Elasticsearch' and specify additional configuration options under `install`.   Or you can create and configure your Jaeger instance and set  `spec:addons:jaeger:name` to the name of the Jaeger instance, for example  `jaeger-production`.
+
+.Control Plane default Jaeger parameters (Elasticsearch)
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: production
+spec:
+  version: v2.0
+  tracing:
+    sampling: 10000
+    type: Jaeger
+  addons:
+    jaeger:
+      name: jaeger
+      install:
+        storage:
+          type: Elasticsearch
+        ingress:
+          enabled: true
+  runtime:
+    components:
+      tracing.jaeger.elasticsearch: # only supports resources and image name
+        container:
+          resources: {}
+----
+
+[id="ossm-deploying-jaeger-streaming_{context}"]
+== Streaming Jaeger deployment
```

Comment:
This configuration is the same as for the "external" jaeger configuration use case.  I don't know if you want to identify it as such, and then just reference it from the streaming jaeger doc.

Reply:
I left it as is.